code/go/0chain.net/blobbercore/allocation/entity.go

```diff
@@ -49,6 +55,21 @@ type Allocation struct {
 	// Ending and cleaning
 	CleanedUp bool `gorm:"column:cleaned_up;not null;default:false"`
 	Finalized bool `gorm:"column:finalized;not null;default:false"`
+
+	// Flag to determine if anyone can extend this allocation
+	ThirdPartyExtendable bool `json:"third_party_extendable" gorm:"column:third_party_extendable;not null;default:false"`
```

Comment:
not sure that we need this field on blobber

Reply:
It's actually not used so yes I'll be removing that.